scss/pages/_home.scss

```
@@ -0,0 +1,195 @@
+section.banner#banner {
```

```suggestion
section.banner {
```
Hạn chế style với selector ID